cfht2caom2/tests/test_hdf5.py

```diff
@@ -0,0 +1,135 @@
+#  any later version.                   toute version ultérieure.
+#
+#  OpenCADC is distributed in the       OpenCADC est distribué
+#  hope that it will be useful,         dans l’espoir qu’il vous
+#  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+#  without even the implied             GARANTIE : sans même la garantie
+#  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+#  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+#  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+#  General Public License for           Générale Publique GNU Affero
+#  more details.                        pour plus de détails.
+#
+#  You should have received             Vous devriez avoir reçu une
+#  a copy of the GNU Affero             copie de la Licence Générale
+#  General Public License along         Publique GNU Affero avec
+#  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+#  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+#                                       <http://www.gnu.org/licenses/>.
+#
+#  : 4 $
+#
+# ***********************************************************************
+#
+
+import glob
+import warnings
+
+from astropy.utils.exceptions import AstropyUserWarning
+from astropy.wcs import FITSFixedWarning
+from mock import patch
+from os.path import basename, dirname,join, realpath
+
+from cadcdata import FileInfo
+from caom2pipe.manage_composable import StorageName, read_obs_from_file
+from cfht2caom2 import CFHTName, COLLECTION
+from cfht2caom2 import fits2caom2_augmentation, reader
+import test_fits2caom2_augmentation
+
+
+THIS_DIR = dirname(realpath(__file__))
+TEST_DATA_DIR = join(THIS_DIR, 'data')
+SITELLE_DIR = join(TEST_DATA_DIR, 'sitelle')
+
+
+def pytest_generate_tests(metafunc):
+    obs_id_list = glob.glob(f'{SITELLE_DIR}/*.hdf5')
+    metafunc.parametrize('test_name', obs_id_list)
+
+
+@patch('cfht2caom2.metadata.CFHTCache._try_to_append_to_cache')
+@patch('caom2pipe.astro_composable.get_vo_table')
+def test_visitor(vo_mock, cache_mock, test_name):
+    warnings.simplefilter('ignore', category=AstropyUserWarning)
+    warnings.simplefilter('ignore', category=FITSFixedWarning)
+    vo_mock.side_effect = test_fits2caom2_augmentation._vo_mock
+    # cache_mock there so there are no update cache calls - so the tests
+    # work without a network connection
+    original_scheme = StorageName.scheme
+    original_collection = StorageName.collection
+    try:
+        StorageName.scheme = 'cadc'
+        StorageName.collection = COLLECTION
+        storage_name = CFHTName(
+            file_name=basename(test_name),
+            instrument=test_fits2caom2_augmentation._identify_inst_mock(None, test_name),
+            source_names=[test_name],
+        )
+        file_info = FileInfo(id=storage_name.file_uri, file_type='application/x-hdf5')
+        import logging
+        logging.error(storage_name)
```

Comment:
maybe move `import logging` in proper place and remove the second one down below.